libyara/filemap.c

```diff
@@ -159,7 +159,7 @@ YR_API int yr_filemap_map_fd(
 
 #ifdef __linux__
 #include <sys/vfs.h>
-#include <linux/magic.h>
+#define PROC_SUPER_MAGIC	0x9fa0
```

Comment:
Can you add a comment here pointing to the github issue? I wasn't to make sure that in the future we known where this constant came from.